scripts/test.mjs

```diff
@@ -0,0 +1,15 @@
+import { spawn } from 'node:child_process';
+
```

Comment:
Can you comment here and link to this PR (https://github.com/mui/material-ui/pull/40430)? We would likely never have set it up this way if we had started out with pnpm and that way it's documented for future generations.

_Another thing we could do at some is add some output saying "`pnpm t <pattern>` is deprecated, use "`pnpm tw <pattern>`" instead. This way of running the tests will be removed in a future version of the MUI core repository". This gives contributors enough time to adapt their muscle memory and us to option of removing the complexity of this script in the future._